### DISCUSSION TITLE
Add automatic "Refund" label in the wallet when an invoice is refunded

### DISCUSSION THREAD

**Discussion by pavlenex** [btcpayserver/btcpayserver#2319] [Ideas / Feature Requests]
**Is your feature request related to a problem? Please describe.**
It would be good to add a "Refund" label when doing a [refund](https://docs.btcpayserver.org/Refund/).

**Describe the solution you'd like**
Add an automatic Refund label similar to "Invoice" or "Payout" inside the wallet.
![Refund](https://user-images.githubusercontent.com/36959754/89321921-d3c22f80-d683-11ea-9932-a18daec93844.PNG)

**Comment by NicolasDorier** [comment]
it is not good first issue. This is quite difficult.
Why payout is not good enough?

**Comment by pavlenex** [comment]
@NicolasDorier I thought replicating similar behavior to invoices would be enough. Removed the label. 
You don't get payout label when issuing a refund. Is that expected behavior or a bug? 

I think refund sounds way more understandable to people than payout. Payout should be for payouts and refund for refund even though it's technically same concept, people don't know this.
